src/lib/htmlInputPropsUtils.js

```diff
@@ -1,5 +1,14 @@
 import _ from 'lodash'
 
+export const htmlInputAria = [
+  'aria-activedescendant', 'aria-atomic', 'aria-autocomplete', 'aria-busy', 'aria-checked', 'aria-controls',
+  'aria-describedby', 'aria-disabled', 'aria-dropeffect', 'aria-expanded', 'aria-flowto', 'aria-grabbed',
+  'aria-haspopup', 'aria-hidden', 'aria-invalid', 'aria-label', 'aria-labelledby', 'aria-level', 'aria-live',
+  'aria-multiline', 'aria-multiselectable', 'aria-orientation', 'aria-owns', 'aria-posinset', 'aria-pressed',
+  'aria-readonly', 'aria-relevant', 'aria-required', 'aria-selected', 'aria-setsize', 'aria-sort', 'aria-valuemax',
+  'aria-valuemin', 'aria-valuenow', 'aria-valuetext', 'role',
+]
+
```

Comment:
How about we update the existing `partitionHTMLInputProps` loop to check for `aria-` prefixed props instead?  If `/^aria-.*$/.test(prop)` then it is automatically an `inputProp` and not a `rest` prop.